src/main/js/modules/blocks.js

```diff
@@ -282,29 +291,64 @@ var blocks = {
   },
   fence: {
     oak: 85,
+    nether_brick: 113,
```

Comment:
We have `slab.netherbrick`, `double_slab.netherbrick`… why not `fence.netherbrick`?